tests/unittests/test_interfaces.py

```diff
@@ -0,0 +1,17 @@
+import torch
+
+
+def test_load_audio(tmpdir):
+    from speechbrain.pretrained.interfaces import Pretrained
+
+    verification = Pretrained.from_hparams(
```

Comment:
Still wondering about loading a 100+MB pretrained model here. I deleted my earlier comments on using a shorter pretrained model. All that is relevant here are these lines

```python
        source, fl = split_path(path)
        path = fetch(fl, source=source, savedir=savedir)
        signal, sr = torchaudio.load(path, channels_first=False)
```

That's what needs to be tested for the fetch in pretrained interfaces.